src/database/DataTypes/InsurancePolicy.js

```diff
@@ -2,7 +2,8 @@ import Realm from 'realm';
 
 export class InsurancePolicy extends Realm.Object {
   get policyNumber() {
-    return `${this.policyNumberPerson} ${this.policyNumberFamily}`;
+    if (!this.policyNumberPerson) return this.policyNumberFamily;
```

Comment:
both values are required in the schema aren't they? 

https://github.com/openmsupply/mobile/blob/d515a274516ae0b1b434add43eb967e4d94f8ad4/src/database/DataTypes/InsurancePolicy.js#L15-L16

🤔

Reply:
Yeah, I believe that just means it can't be set to `null` or `undefined`? As it's a primitive I think it makes more sense to have it initialised to a falsey but non-null value? 

To make it more explicit can do:

```
policyNumberPerson: { type: 'string', default: '' };
```

but I think the above is equivalent to the existing declaration?

Reply:
I didn't know so I did a quick test with:

```
UIDatabase.write(() => {
      const insuranceProvider = UIDatabase.update('InsuranceProvider', {
        id: '1',
      });

      UIDatabase.update('InsurancePolicy', { id: '1', insuranceProvider });
    });
```

And got the error "Missing value for property `InsurancePolicy.policyNumberFamily`" - so I believe you need to explicitly set the default. Also doesn't say anything in the docs about defaults being set for primitives automatically: https://realm.io/docs/javascript/latest/#default-property-values

Reply:
Oops, looks like I misinterpreted the examples in the [docs](https://realm.io/docs/javascript/latest/#models) (I guess that includes implicitly on construction).

eeb4b8c36cbd900662740f69499455cbffe7d945.